src/shared/Atlassian.Bitbucket.Tests/BitbucketHostProviderTest.cs

```diff
@@ -46,6 +51,22 @@ public void BitbucketHostProvider_IsSupported(string protocol, string host, bool
             Assert.Equal(expected, provider.IsSupported(input));
         }
 
+        [Fact]
+        public void BitbucketHostProvider_IsSupported_FailsForNullInput()
+        {
+            InputArguments input = null;
+            var provider = new BitbucketHostProvider(new TestCommandContext());
+            Assert.Equal(false, provider.IsSupported(input));
```

Comment:
Xunit CI analysers are complaining about `Assert.Equal(false, X)`.. we should use `Assert.False(X)`:

```suggestion
            Assert.False(provider.IsSupported(input));
```